GitUI/Editor/FileViewerInternal.cs

```diff
@@ -73,6 +73,31 @@ public FileViewerInternal()
             VRulerPosition = AppSettings.DiffVerticalRulerPosition;
         }
 
+        /// <summary>
+        /// Clean up any resources being used.
+        /// </summary>
+        /// <param name="disposing">true if managed resources should be disposed; otherwise, false.</param>
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                _diffHighlightService = null;
+                _lineNumbersControl = null;
+                _findAndReplaceForm?.Dispose();
+                DoubleClick = null;
+                KeyUp = null;
+                MouseLeave = null;
+                MouseEnter = null;
+                MouseMove = null;
+                SelectedLineChanged = null;
+                EscapePressed = null;
```

Comment:
😕 How would any of these get invoked after the control is disposed? I'm not sure what the value in unhooking them is.

Reply:
Removed.